.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+omit=*apps.py
```

Comment:
let me know if there is a better way of doing this than creating a `.coveragerc`

Reply:
There's a coverage section within `tox.ini`. You should be able to add the config you want there. We do something similar with `setup.cfg` in platform ([link](https://github.com/pennlabs/platform/blob/05e035c8ab0703aa34971ffed1819641b9783721/backend/setup.cfg#L15))